src/App.js

```diff
@@ -1,13 +1,32 @@
-
+import { useEffect, useState } from "react";
+import Card from "./components/ui/Card";
 const App = () => {
-  function message(e){
-    e.preventDefault();
-    alert("clicked!");
+  const [data, cardData] = useState([]);
```

Comment:
it is  a convention in react to start your state updater function with `set`
```suggestion
  const [data, setData] = useState([]);
```